make-image.lisp.in

```diff
@@ -18,4 +18,5 @@
                                                 (stumpwm:stumpwm)
                                                 0)
                                     :executable t
-                                    :purify t)
+                                    :purify t
+                                    #+sb-core-compression :compression #+sb-core-compression t)
```

Comment:
Instead of feature expressions which have to be repeat for each form, what do you think of?

```suggestion
                                    :compression (if (member :sb-core-compression *features*) t nil)
```

Note We have to pass t or nil to compression because its type is expected to satisfy the following `(check-type compression (or boolean (integer -1 9)))`. `t` is equivalent to -1, that is the default compression which is level 6 afaik.

Reply:
I wasn't sure if the ```:compression``` keyword was valid with a SBCL compiled without compression support, but apparently is it, so I pushed your suggestion because indeed it looks nicer.